mmdet3d/core/visualizer/image_vis.py

```diff
@@ -132,6 +117,8 @@ def draw_lidar_bbox3d_on_img(bboxes3d,
     pts_2d[:, 1] /= pts_2d[:, 2]
     imgfov_pts_2d = pts_2d[..., :2].reshape(num_bbox, 8, 2)
 
+    return plot_rect3d_on_img(img, num_bbox, imgfov_pts_2d, color, thickness)
+    '''
```

Comment:
Why keep the comments?

Reply:
Just for debug purposes. I am deleting them now.